plugins/bitbucket-cloud-common/bitbucket-cloud.oas.json

```diff
@@ -8507,6 +8507,58 @@
         }
       ]
     },
+    "/repositories/{workspace}/{repo_slug}/pipelines-config/caches": {
```

Comment:
This will be stripped out once I reduce the models. I left everything in at this point as it was a draft.

On a side note, one of the OpenAPI models breaks the backstage pipeline. Not recalling which one right now